### ISSUE TITLE
Draft Create Auto Detect pom.xml file during Java detection

### ISSUE THREAD

**Issue by msftph** [Azure/draft#223]
**Is your feature request related to a problem? Please describe.**

When running `draft create` on the following folder https://github.com/buildpacks/samples/blob/main/apps/java-maven/, draft shows the following prompt:

```
[Draft] --- Detecting Language ---
Use the arrow keys to navigate: ↓ ↑ → ←
? Linguist detected Java, are you using maven or gradle?:
    gradle
  > maven
```

**Mention what platform you want to support the new feature**

OSS draft

**Describe the solution you'd like**

Draft should detect that maven is being used by seeing the pom.xml file in the repository. 

**Describe alternatives you've considered**

(none)

**Additional context**

(none)

**Comment by davidgamero** [comment]
thanks for the suggestion! this is a great point

we probably can add a more provider-based approach, but a simple check in [here](https://github.com/davidgamero/draft/blob/ec4a40b61c24718deab20ed2762aefd396606fb2/cmd/create.go#L200-L200) would do it using the RepoReader to test if a pom.xml exists

what do you think @hsubramanianaks ?

**Comment by hsubramanianaks** [comment]
@davidgamero Yep agree we can detect (look for pom.xml or *.gradle using repo reader) before prompting to users.

**Comment by brunoborges** [comment]
Some projects may contain both Gradle and Maven build files. It is unlikely in the majority of cases, but for sample applications written by framework authors, it is quite common. Due to this reason, I would like to propose Maven to be considered the default auto discovery choice when both are present.
